yarn.lock

```diff
@@ -4753,6 +4753,14 @@
   dependencies:
     tippy.js "^6.3.1"
 
+"@trezor/rollout@1.1.0":
+  version "1.1.0"
```

Comment:
Shouldn't this be 1.2.0? :thinking:

Reply:
this is not direct dependency but is added via connect instead. `yarn why @trezor/rollout`

```
=> Found "@trezor/rollout@1.1.0"
info Has been hoisted to "@trezor/rollout"
info Reasons this module exists
   - "workspace-aggregator-f81debb6-c946-4418-b164-aecc50eb92de" depends on it
   - Specified in "dependencies"
   - Hoisted from "_project_#@trezor#rollout"
   - Hoisted from "_project_#@trezor#suite#trezor-connect#@trezor#rollout"
info Disk size without dependencies: "260KB"
info Disk size with unique dependencies: "956KB"
info Disk size with transitive dependencies: "1.11MB"
info Number of shared dependencies: 3
```

So I think this does not matter. Once connect is in monorepo this will not be happening.

Reply:
Eh. Ok..